dp2/config.yml

```diff
@@ -1,17 +1,12 @@
-#WS CONFIGURATION
+#WS CONFIGURATION if app is not available
```

Comment:
What does "if the app is not available" mean here?

Reply:
I was thinking to use the WS configuration if app is not available in path or through the app_path variable.

(but from your comment in the argument test to launch the app, that would need to be change)